### PR TITLE
Add instructions to enable stork initializer

### PR DESCRIPTION
**Pull request by disrani-px**

### CODE REVIEW

File: scheduler/kubernetes/scheduler-convergence.md [hunk 1]
```diff
@@ -53,3 +56,15 @@ spec:
...
+If you are not able to update the schedulerName for you applications to use
+stork, you can enable the app-initializer feature. This uses the Kubernetes
+[AdmissionController Initializer](https://kubernetes.io/docs/admin/extensible-admission-controllers/#initializers)
+feature to automatically update the scheduler to stork if your application
+(deployment or statefulset) is using volumes backed by Portworx.
+
+To enable the Initializer you need to:
+* [Enable the Intializer feature in your Kubernetes cluster](https://kubernetes.io/docs/admin/extensible-admission-controllers/#enable-initializers-alpha-feature) since it is an alpha feature in Kubernetes.
+* Add "--app-initializer=true" option in the stork deployment
+* Add the [stork-initializer spec](https://raw.githubusercontent.com/libopenstorage/stork/master/specs/stork-initializer.yaml) to you Kubernetes cluster using `kubectl create -f stork-initializer.yaml`
```
Comment (harsh-px) [inline scheduler/kubernetes/scheduler-convergence.md]: to your Kubernetes ...

Comment (harsh-px) [inline scheduler/kubernetes/scheduler-convergence.md]: change `kubectl create` to `kubectl apply` to avoid warnings on subsequent applies.

Comment (disrani-px) [reply scheduler/kubernetes/scheduler-convergence.md]: Done

Comment (disrani-px) [reply scheduler/kubernetes/scheduler-convergence.md]: Done

**Pull request merged**